thirdparty/libsimplewebm/OpusVorbisDecoder.cpp

```diff
@@ -72,6 +72,51 @@ bool OpusVorbisDecoder::isOpen() const
 	return (m_vorbis || m_opus);
 }
 
+bool OpusVorbisDecoder::getPCMS(WebMFrame &frame, float *buffer, int &numOutSamples)
```

Comment:
Could you contribute that change upstream too so that we don't diverge? https://github.com/zaps166/libsimplewebm
cc @zaps166

Reply:
Ok, but shouldn't be e.g. "getPCMF" (S16 -> signed 16 bit; F -> float)?

Reply:
This is the one of parts I copied from the original author's branch and I overlooked function naming.
Yes, I think it should be getPCMF for meaning it's float version.
I'll update that.

About upstream, I didn't think about that but I'm happy to prepare PR for libsimplewebm too.